packages/api/src/hooks/useCountryConfig.ts

```diff
@@ -0,0 +1,303 @@
+import { useCallback, useMemo } from 'react';
+import useFetch from '../useFetch';
+
+type TCountryCode = Uppercase<
+    | 'af'
+    | 'ax'
+    | 'al'
+    | 'dz'
+    | 'as'
+    | 'ad'
+    | 'ao'
+    | 'ai'
```

Comment:
IMO, we shouldn't be maintaining the list of country codes in FE. The type of the country code should be just a string.